tasks/office.py

```diff
@@ -33,30 +34,36 @@ class OfficeDataStorageMixin:
     """
 
     DATA_LIST_MAX: int = 100
-    local_store: Dict[str, list] = {
-        MOVE_TASK_ORDER: [],
-        MTO_SHIPMENT: [],
-        MTO_SERVICE_ITEM: [],
-        PAYMENT_REQUEST: [],
+    local_store: Dict[str, Dict[str, Dict]] = {
+        MOVE_TASK_ORDER: {},
+        MTO_SHIPMENT: {},
+        MTO_SERVICE_ITEM: {},
+        ORDER: {},
+        PAYMENT_REQUEST: {},
     }  # data stored will be shared among class instances thanks to mutable dict
 
-    def get_stored(self, object_key, *args, **kwargs):
+    def get_stored(self, object_key, object_id=None, *args, **kwargs):
         """
         Given an object_key that represents an object type from the MilMove app, returns an object of that type from the
         list.
 
-        :param object_key: str in [MOVE_TASK_ORDER, MTO_SHIPMENT, MTO_SERVICE_ITEM, PAYMENT_REQUEST]
+        :param object_key: str in [MOVE_TASK_ORDER, MTO_SHIPMENT, MTO_SERVICE_ITEM, ORDER, PAYMENT_REQUEST]
+        :param object_id: str uuid of a single object item key to return
         """
         data_list = self.local_store[object_key]
+        data_item = data_list.get("object_id")
+        if data_item is not None:
+            return data_item
```

Comment:
Should this be `data_item = data_list.get(object_id)`? This is trying to get an item with the literal key `"object_id"` instead of whatever value the variable `object_id` has. I was under the impression the shape of this dict would be like:
```python
{
    "uuid1": {<object data here>},
    "uuid2": {<object 2 data here>},
}
```
but I may have misunderstood.